RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -5357,7 +5030,8 @@ boolean auto_is_valid(item it)
 
 boolean auto_is_valid(familiar fam)
 {
-	return bees_hate_usable(fam.to_string()) && glover_usable(fam.to_string()) && is_unrestricted(fam);
+	familiar hundo = to_familiar(get_property("auto_100familiar"));
```

Comment:
THANK YOU

although I would *love* a log (maybe `auto_debug_print()`) that logged when this function returns false *because* of a 100% run.